modules/network-observability-packet-drops.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-packet-drops_{context}"]
+= Working with packet drops
+Packet loss occurs when one or more packets of network flow data fail to reach their destination. You can track these drops by editing the `FlowCollector` to the specifications in the following YAML example.
++
+[IMPORTANT]
+====
+CPU and memory usage increases when this feature is enabled.
+====
+
+.Prerequisites
+* Access to an {product-title} cluster with version 4.13. 
+* Kernel supported by Red Hat Enterprise Linux (RHEL) 9.2.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-pkt-drop_{context}"]
+.Configure `FlowCollector` for packet drops
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features:
+       - PacketsDrop            <1>                         
+      privileged: true       <2>
+----
+<1> You can start reporting the packet drops of each network flow by listing the `PacketsDrop` parameter in the `spec.agent.ebpf` specification list.   
+<2> Note that the `spec.agent.ebpf.privileged` specification value must be `true` for packet drop tracking to be enabled.
```

Comment:
No need of `packet drop tracking to be enabled.` It can just be  `Note that the `spec.agent.ebpf.privileged` specification value must also be set to `true``

Reply:
Addressed

Reply:
@Amoghrd I changed the text to "Note that the `spec.agent.ebpf.privileged` specification value must be `true` for packet drop tracking."

Reply:
Discussed with Amogh to add "Additionally" here.